cpp/tensorrt_llm/kernels/cutlass_kernels/python/generate_kernels.py

```
@@ -273,8 +270,7 @@ def instantiate_operation_sm80(operation):
 
     instantiation = f"""
             template void sm80_generic_fused_moe_gemm_kernelLauncher<{act_tag}, {weight_tag}, {operation.cta_shape[0]}, {operation.cta_shape[1]}, {operation.cta_shape[2]}, {operation.stage}, {epi_tag}>
```

Ruff (0.12.2)

Line too long (201 > 120)

(E501)